Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java

```diff
@@ -430,7 +430,7 @@ public boolean onMenuItemClick(MenuItem item) {
                         switch (item.getItemId()) {
```

Comment:
Suggestion (💡): Consider extracting switch case duplicate code to prevent people from making update mistakes. For example:
```
switch (item.getItemId()) {
    case R.id.sort_alphabetical:
        updateSortOrder("alphabetical_az", ALPHABETICAL_ASCENDING);
        return true;
    case R.id.sort_alphabetical_reverse:
        updateSortOrder("alphabetical_za", ALPHABETICAL_DESCENDING);
        return true;
    case R.id.sort_newest_created:
        updateSortOrder("created_newest", DATE_CREATED_DESCENDING);
        return true;
    case R.id.sort_oldest_created:
        updateSortOrder("created_oldest", DATE_CREATED_ASCENDING);
        return true;
    case R.id.sort_newest_modified:
        updateSortOrder("modified_newest", DATE_MODIFIED_DESCENDING);
        return true;
    case R.id.sort_oldest_modified:
        updateSortOrder("modified_oldest", DATE_MODIFIED_ASCENDING);
        return true;
    default:
        return false;
}

private void updateSortOrder(String sortOrderLabel, int prefSortOrder) {
    AnalyticsTracker.track(
            AnalyticsTracker.Stat.LIST_SORTBAR_MODE_CHANGED,
            AnalyticsTracker.CATEGORY_SETTING,
            sortOrderLabel
    );
    mPreferences.edit().putString(PrefUtils.PREF_SORT_ORDER,
            String.valueOf(prefSortOrder)
    ).apply();
    refreshList();
}
```

PS: I understand that this is not part of the PR change, so please ignore my comment if you feel the same.

Reply:
I added the `updateSortOrder` method in a23b117.